src/Components/PresentationalComponents/DownloadReportKebab/KebabItems.js

```diff
@@ -4,31 +4,33 @@ import messages from '../../../Messages';
 import { FormattedMessage } from 'react-intl';
 
 export const kebabItemDownloadJSON = (downloadReport, { ...props }) => (
-    <DropdownItem key="json" component="button" onClick={() => downloadReport('json')} {...props}>
+    <DropdownItem key="json" component="button" onClick={() => downloadReport('json')} {...props} ouiaId="download_json">
```

Comment:
not working, ther is `data-ouia-component-id="OUIA-Generated-DropdownItem-1"`

Reply:
Hi, @psegedy can you please provide me details for this?
I am not sure where your pointing.

Reply:
@mayurilahane try it locally, navigate to dropdown item that should use `download_json` ouiaId and notice that the component doesn't have this ID but generated one.